From fdf5b1e5582acae15d485a4c2e2af2ff2819a8c6 Mon Sep 17 00:00:00 2001
From: Jan Zeleny <jzeleny@redhat.com>
Date: Tue, 29 Jul 2014 15:36:16 +0200
Subject: [PATCH] Remove sclbuild, as it's not that useful

---
 Makefile | 5 ++---
 sclbuild | 8 --------
 2 files changed, 2 insertions(+), 11 deletions(-)
 delete mode 100755 sclbuild

diff --git a/Makefile b/Makefile
index f050285c1ca85537f30bcfc7bf2cd0a38dd75821..a2d368926638c13ba3b230f02f17e27229046445 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ VERSION=`date +%Y%m%d`
 WARNINGS?=-Wall -Wshadow -Wcast-align -Winline -Wextra -Wmissing-noreturn
 CFLAGS?=-O2
 CFILES=scl.c
-OTHERFILES=Makefile scl_enabled macros.scl scl.1 scldeps.sh scl.attr brp-scl-compress brp-scl-python-bytecompile scl.bash sclbuild
+OTHERFILES=Makefile scl_enabled macros.scl scl.1 scldeps.sh scl.attr brp-scl-compress brp-scl-python-bytecompile scl.bash
 SOURCES=$(CFILES) $(OTHERFILES)
 
 BINDIR?=/usr/bin
@@ -37,7 +37,6 @@ install: all
 	mkdir -p $(DESTDIR)/$(RPMCONFDIR)/fileattrs
 	cp macros.scl $(DESTDIR)/$(CNFDIR)/rpm
 	cp scl $(DESTDIR)/$(BINDIR)
-	cp sclbuild $(DESTDIR)/$(BINDIR)
 	cp scl_enabled $(DESTDIR)/$(BINDIR)
 	cp scl.1 $(DESTDIR)/$(MANDIR)/man1
 	cp scl.attr $(DESTDIR)/$(RPMCONFDIR)/fileattrs
@@ -47,7 +46,7 @@ install: all
 	cp scl.bash $(DESTDIR)/$(CNFDIR)/bash_completion.d
 
 uninstall:
-	rm -f $(BINDIR)/scl $(BINDIR)/scl_enabled $(BINDIR)/sclbuild
+	rm -f $(BINDIR)/scl $(BINDIR)/scl_enabled
 	rm -f $(CNFDIR)/rpm/macros.scl
 	rm -f $(MANDIR)/man1/scl.1
 	rm -f $(RPMCONFDIR)/fileattrs/scl.attr
diff --git a/sclbuild b/sclbuild
deleted file mode 100755
index 183d8e8d8fbcd8140052cf9b381ae73bc9f633c1..0000000000000000000000000000000000000000
--- a/sclbuild
+++ /dev/null
@@ -1,8 +0,0 @@
-#!/bin/sh
-if [ $# -le 1 ]; then
-	echo "$0 <collection name> <rpmbuild arguments>"
-else
-	SCL=$1
-	shift
-	rpmbuild $@ --define "scl $SCL"
-fi
-- 
1.9.3

